docs/pages/reference/cli.mdx

```diff
@@ -2586,6 +2586,65 @@ $ tctl alerts create \
     "The system is under maintenance, functionality may be limited."
 ```
 
+### tctl alerts list
+
+Lists cluster alerts. This command can also be invoked as `tctl alerts ls`.
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--verbose` (`-v`) | false |boolean | If set, display detailed alert info |
+| `--labels` | none | any | A list of labels to filter by |
```

Comment:
Would it make sense to list the allowed value of `--labels` as "string" since that's what a label would be?

Reply:
Good call. I will match the same prose we use for other commands that accept `--labels`